### ISSUE TITLE
[Feature Request] See favorite projects in sidebar

### ISSUE THREAD

**Issue by jboulter11** [getsentry/sentry#58038]
### Problem Statement

I would like to see my favorited projects in my sidebar so it's one fewer click to be able to get to them.

It takes me two clicks to get back to my project now.  Once on `Projects` then once on the project itself.

![image](https://github.com/getsentry/sentry/assets/6633472/3aa5615f-eedb-4c86-975b-b3fd76f5e865)

### Solution Brainstorm

I like the way the Performance tab has a dropdown for queries.  I'd like it to look like that.

### Product Area

Projects - Project Details

**Comment by ceorourke** [comment]
Thanks for the feature request @jboulter11! I put it in our backlog for now and am tagging our product manager.

cc @rachrwang seems like a small qol improvement

**Comment by rachrwang** [comment]
@jboulter11 - thank you for sharing this feature request! I'd love to learn more about how you use the project pages today - some questions that are top of mind include:

- What do you use the project pages for today, i.e. what's most valuable/ critical for your workflow?
- How many projects do you engage with on a regular basis?

**Comment by jboulter11** [comment]
I guess I do usually use it to get to issues.  I am not super used to the Sentry UI yet, so I don't think to go to Issues first, I think to go to the project first.  Having favorited projects on issues in the side bar might also make sense. 🤷 

Two projects on a regular basis that I'm switching between.

**Comment by ceorourke** [comment]
@jboulter11 depending on your plan (I believe selecting multiple projects is on the Business plan only) you could [set a default search](https://docs.sentry.io/product/reference/search/saved-searches/#set-a-default-search) with your favorite projects so when you load the issues stream it's already filtering to the projects you care about the most.

**Comment by jboulter11** [comment]
That's helpful to know.  I'm also flipping between settings pages to check configurations between multiple projects, but I'm realizing now that there's a dropdown to quick switch between them in the settings screen.

**Comment by ceorourke** [comment]
Yep! You can also use the command palette (command + k) to quickly navigate around if that's helpful for you.

**Comment by ceorourke** [comment]
There doesn't seem to be, I'm guessing because it's the main landing page and available in the sidebar. I can put in a feature request for you if you'd like though.

**Comment by jboulter11** [comment]
I'd appreciate that, thank you!

**Comment by ceorourke** [comment]
I added it here: https://github.com/getsentry/sentry/issues/58350 feel free to subscribe for updates.

**Comment by gaprl** [comment]
Hey @jboulter11, this feature request was opened over a year ago. Is this still something you'd find valuable? We're reviewing older feature requests and want to ensure we're focusing on features that are still needed. If we don't hear back in the next couple of weeks, we may close this issue to keep our backlog focused on active requests.

**Comment by jboulter11** [comment]
I don't think it makes sense as-is, given the sidebar redesign.  The core problem still exists that I have to do multiple clicks to see different projects rather than being able to tab between them more easily with only one click, but I can at least make custom views now, and the starred projects are shown in the Insights tab which I can get to by mousing over insights then clicking the project.  You can close this, there are more important things I'd like to see in Sentry than this.
